src/pray.c

```diff
@@ -1794,7 +1794,7 @@ dosacrifice(void)
                                   hcolor((u.ualign.type == A_LAWFUL)
                                             ? NH_GOLDEN
                                             : u.ualign.type
-                                               ? NH_RED
+                                               ? NH_BLACK
```

Comment:
This is obsoleted by a commit I made on the savebreaking branch (altar colors are changed back to vanilla's and therefore the glow colors should be too).